go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/Ptt-official-app/go-openbbsmiddleware
 go 1.16
 
 require (
-	github.com/Ptt-official-app/go-pttbbs v0.19.1
+	github.com/Ptt-official-app/go-pttbbs v0.19.2
```

Comment:
no need to change the version in this pr though.

I prefer to have a separated PR specifically for version update.